[Observability Onboarding] Copy update for landing page messages

AlejandroFrndz opened the pull request:
Closes #241747 
## Summary

Updates messages in observability onboarding landing page when metrics onboarding is enabled.

<img width="1236" height="772" alt="Screenshot 2025-11-04 at 11 44 03" src="https://github.com/user-attachments/assets/75d9b5a0-e9f0-4e00-ac8d-99c5fdb85775" />



<!--ONMERGE {"backportTargets":["8.19","9.1","9.2"]} ONMERGE-->

AlejandroFrndz committed:
fix: Update messages on obs onboarding landing page

AlejandroFrndz committed:
fix: Update README to use parallel.playwright.config.ts

Updates README examples to reference the renamed Playwright config file (parallel.playwright.config.ts) instead of the underscore variant. This corrects the example test commands for ESS and Serverless runs and prevents confusion when invoking tests.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/358067)
* Commit: f9829c68f5525aba24e1e7c366b6ad974683c7a8

### Failed CI Steps
* [Jest Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/358067#019a4e9c-be9b-491b-a961-faa1c59eaef0)
* [Jest Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/358067#019a4eb5-6a74-4257-b444-de326db5633e)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358067#019a4e9c-be9b-491b-a961-faa1c59eaef0) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358067/jobs/019a4e9c-be9b-491b-a961-faa1c59eaef0/artifacts/019a4eb5-56fc-47e8-910a-8a816662c109) Jest Tests #<span></span>7 / OnboardingFlowForm Common Elements should always show Cloud tile with same description
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358067#019a4eb5-6a74-4257-b444-de326db5633e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358067/jobs/019a4eb5-6a74-4257-b444-de326db5633e/artifacts/019a4ecf-d46f-43ab-bf67-b3e3dd393f9b) Jest Tests #<span></span>7 / OnboardingFlowForm Common Elements should always show Cloud tile with same description
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358067#019a4e9c-be9b-491b-a961-faa1c59eaef0) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358067/jobs/019a4e9c-be9b-491b-a961-faa1c59eaef0/artifacts/019a4eb5-56fb-4403-aaff-c1b9ff0843d2) Jest Tests #<span></span>7 / OnboardingFlowForm Complete Tier (Metrics Onboarding Enabled) should show complete tier description for Host
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358067#019a4eb5-6a74-4257-b444-de326db5633e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358067/jobs/019a4eb5-6a74-4257-b444-de326db5633e/artifacts/019a4ecf-d46f-4f5b-a03c-11e3febb2652) Jest Tests #<span></span>7 / OnboardingFlowForm Complete Tier (Metrics Onboarding Enabled) should show complete tier description for Host
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358067#019a4e9c-be9b-491b-a961-faa1c59eaef0) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358067/jobs/019a4e9c-be9b-491b-a961-faa1c59eaef0/artifacts/019a4eb5-56fd-434c-b0f1-89152ef9c5a6) Jest Tests #<span></span>7 / OnboardingFlowForm Complete Tier (Metrics Onboarding Enabled) should show complete tier description for Kubernetes
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/358067#019a4eb5-6a74-4257-b444-de326db5633e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/358067/jobs/019a4eb5-6a74-4257-b444-de326db5633e/artifacts/019a4ecf-d46f-48db-ab5a-cb10e1ba59a8) Jest Tests #<span></span>7 / OnboardingFlowForm Complete Tier (Metrics Onboarding Enabled) should show complete tier description for Kubernetes

cc @AlejandroFrndz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/358067","number":358067,"commit":"f9829c68f5525aba24e1e7c366b6ad974683c7a8"}],"number":358067}
buildkite-pr-comment-->

AlejandroFrndz committed:
fix: Updates onboarding flow test copy

Updates onboarding flow form tests to match revised UI wording for onboarding hints.

Adjusts expected strings for host/services SLO messaging, Kubernetes/container monitoring, and cloud telemetry so tests remain aligned with the updated copy and avoid false failures.

gbamparop reviewed (approved):
That was fast 🚀 , thanks!

kibanamachine commented:
Starting backport for target branches: 8.19, 9.1, 9.2

https://github.com/elastic/kibana/actions/runs/19076016947
